pkg/nfd-worker/nfd-worker.go

```diff
@@ -444,7 +464,7 @@ func (w *nfdWorker) configure(filepath string, overrides string) error {
 			if err != nil {
 				return fmt.Errorf("Failed to parse config file: %s", err)
 			}
-			stdoutLogger.Printf("Configuration successfully loaded from %q", filepath)
+			klog.Infof("Configuration successfully loaded from %q", filepath)
```

Comment:
pretty much every log is with lower case, why this one is upper?

Reply:
this will become a TODO as per discussion with @marquiz over slack

Reply:
Historical reasons. I tried to alter the actual content of the log messages as little as possible in this PR. We can standardize these in a follow-up PR- WDYT?